app/src/main/res/values/strings.xml

```diff
@@ -211,5 +211,11 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="bug">Bug</string>
     <string name="input"> Please fill in the input fields and/or assign a rating</string>
     <string name="feedback_thank">Thank you for your feedback!</string>
+    <string name="mentee_with_column">Mentee:</string>
```

Comment:
I think this should be `colon` not `column` :-) https://en.wikipedia.org/wiki/Colon_(punctuation)